guizero/base.py

```diff
@@ -406,11 +406,20 @@ def _close_window(self):
     def update(self):
         self.tk.update()
 
+    def full_screen(self, keybind="<Escape>"):
+        """Make this window full screen and bind the Escape key (or given key) to exit full screen mode"""
+        self.tk.attributes("-fullscreen", True)
+        self.tk.bind(keybind, self.windowed)
```

Comment:
The event manager should be used to bind the escape key. 

The event manager was introduced to get around the constraint that tk can only bind 1 function to 1 event. So if we were to leave it like this a user wouldn't be able to use `<Escape>` for anything else.

Setting an event with the event manager is pretty simple though:

```
self.events.set_event("<FullScreen.Escape>", "<Escape>", self.windowed)
```

The params are:
1. a unique identifier for this event - it can be anything, its just how you reference this particular event.
2. the tk event
3. the function to fun